src/parsers/newTransaction.ts

```diff
@@ -58,31 +55,11 @@ export const parseNewTransaction = async (
     network && isL2Network(network)
       ? { amount: '', display: '' }
       : convertAmountAndPriceToNativeDisplay(amount ?? 0, assetPrice, nativeCurrency);
-  const hash = txHash ? `${txHash}-0` : null;
```

Comment:
this some dumb shiz we were doing previously so split up changes inside of txs so we could show everything